docs/source/customizations/native.mdx

```diff
@@ -281,6 +281,22 @@ tracing::info!(
 );
 ```
 
+### Add custom spans
+<Note>
+
+Make sure to [enable OpenTelemtry tracing](../configuration/telemetry/exporters/tracing/overview) in your configuration if you want to custom traces generated and linked by the Router.
+
+</Note>
+
+To create custom spans and traces you can use the [tracing macros](https://docs.rs/tracing/latest/tracing/index.html#macros) to generate a span.
+
+```rust
+use tracing::{Level, span};
+
+span!(target: "app_spans", Level::TRACE, "my span");
```

Comment:
It's probably best to use info_span! as the level directly corresponds to the log level which most people will set to info.